main.py

```diff
@@ -6,23 +6,32 @@
 # Global variable used for storing data (bad practice)
 data = []
 
+
 # Inefficient data manipulation
 @app.post("/add_item")
 def add_item(item: str):
     global data
     data.append(item)
     return {"message": "Item added successfully"}
 
+
 # Insecure API endpoint with missing authentication
 @app.get("/get_items")
 def get_items():
     return data
 
+
+@app.get("/get_item/{item_id}")
+def get_item(item_id: int):
+    return data[item_id]
+
+
 # Lack of input validation
 @app.post("/divide")
 def divide_numbers(dividend: float, divisor: float):
     return {"result": dividend / divisor}
 
+
 # Inconsistent naming and lack of documentation
 @app.get("/fetch-data")
 def fetchData():
```

Comment:
**Suggestion:** Use consistent naming conventions for function names. The function name 'fetchData' should be in snake_case as 'fetch_data'.
```suggestion
@app.get("/fetch-data")
def fetch_data():
```